Wox/MainWindow.xaml

```diff
@@ -55,23 +55,32 @@
     </Window.InputBindings>
     <Border Style="{DynamicResource WindowBorderStyle}" MouseDown="OnMouseDown" >
         <StackPanel Orientation="Vertical">
-            <TextBox Style="{DynamicResource QueryBoxStyle}"
+            <Grid>
+                <TextBox Style="{DynamicResource QueryBoxStyle}"
+                         Text="{Binding QueryTextSuggestion, Mode=TwoWay, UpdateSourceTrigger=PropertyChanged}"
```

Comment:
why do we need this two way?

Reply:
Because we also display suggestion when users move the highlighter in the result list with the up and down keyboard arrow which changes the SelectedItem property